README.md

```diff
@@ -5,4 +5,9 @@ Coda Packs allow you to extend Coda by building new building blocks that can ope
 To learn more, see [our SDK documentation](https://coda.io/packs/build) or join the [Packs Studio Beta](https://coda.io/packsbeta).
 
 ## Publishing Changes Process
-Adjustments to the `CHANGELOG.md` file should be marked under `### Unreleased` until a release commit is made that updates `package.json` and `CHANGELOG.md` file with a new, later version, and publishes the new version to NPM using `make npm-publish`.
+
+Adjustments to the `CHANGELOG.md` file should be marked under `### Unreleased` until a release commit is made that updates `package.json` and `CHANGELOG.md` file with a new, later version, and publishes the new version to NPM using `make npm-publish`. 
+
+### CHANGELOG
+Our `CHANGELOG.md` file follows the 
```

Comment:
is this line redundant? or merge conflict

Reply:
oh merge conflict thanks